src/flownet/realization/_schedule.py

```diff
@@ -187,6 +187,9 @@ def _calculate_wconhist(self):
                         oil_rate=value["WOPR"],
                         water_rate=value["WWPR"],
                         gas_rate=value["WGPR"],
+                        oil_total=value["WOPT"],
```

Comment:
Recalculating could potentially mean that one doesn't have to supply this at all.